.github/workflows/cmake_builds.yml

```
@@ -119,13 +119,11 @@ jobs:
         cmake  ${CMAKE_OPTIONS} -DCSHARP_MONO=ON -DCMAKE_INSTALL_PREFIX=$GITHUB_WORKSPACE/install-gdal -DCMAKE_UNITY_BUILD=${CMAKE_UNITY_BUILD} -DCMAKE_C_FLAGS=-Werror -DCMAKE_CXX_FLAGS=-Werror ..
         cmake --build $GITHUB_WORKSPACE/superbuild/build --target install -- -j$(nproc)
         ctest -V -R "^csharp.*"
-    - name: Standalone Python bindings build and gdal-utils package
+    - name: Standalone Python bindings build <s>and gdal-utils package</s>
```

I guess this should be
```suggestion
    - name: Standalone Python bindings build
```

---

Oops, that one slipped through, thanks.